src/app/public/modules/action-hub/types/action-hub-data.ts

```diff
@@ -0,0 +1,9 @@
+import { SkyLink } from '../../link-list/types/link';
+import { SkyActionHubNeedsAttention } from '../../needs-attention/types/action-hub-needs-attention';
+
+export type SkyActionHubData = {
```

Comment:
Minor - in most cases where we have things like this we designate them as an `interface` instead of a `type`.

Functionally I don't believe this will have any difference - I'm just looking at that from a consistency standpoint.